design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -359,3 +371,7 @@ type NegotiableQuoteUser @doc(description: "A limited view of a Buyer or Seller
 type StoreConfig {
     is_negotiable_quote_active: Boolean @doc(description: "Indicates if negotiable quote functionality is enabled.")
 }
+
+type SetNegotiableQuoteBillingAddressOutput {
+    quote: NegotiableQuote
```

Comment:
I believe we should add billing_address field  in the NegotiableQuote object if we are only returning quote to match with the proposal in [this PR](https://github.com/magento/architecture/pull/495/files) or add it as an additional output field in the SetNegotiableQuoteBillingAddressOutput.
`billing_address: BillingCartAddress`

Reply:
added

Reply:
```suggestion
    quote: NegotiableQuote!
```